reinforcement_learning/start_train.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Can we merge this into a single script with a flag ? Example: sample.sh --train.

Reply:
merged in 6fa4f2bc5f73ae4d38c9ebe09f6d16e2604ad4a1